### PR TITLE
[ui] Add pagination to Individuals table 

### PR DESCRIPTION
**Pull request by evamillan**
Adds pagination to `IndividualsTable` and embeds the component into `App`. It also adds the `getPaginatedIndividuals` query to retrieve the paginated data needed for the table and tests it.

### CODE REVIEW

File: ui/src/components/IndividualsTable.vue [hunk 1]
```diff
@@ -35,23 +46,102 @@ export default {
   components: {
     IndividualEntry,
     ExpandedIndividual
- },
+  },
   props: {
-    individuals: {
-      type: Array,
+    query: {
```
Comment (sduenas) [inline ui/src/components/IndividualsTable.vue:51]: Can we change the name of this? It's not really a query and it's not querying, so it's a bit confusing. Maybe we can rename it to `fetchPage` or `fetch`? I'm not sure what the conventions are for naming props functions in Vue.

File: ui/src/components/IndividualsTable.vue [hunk 2]
```diff
@@ -35,23 +46,102 @@ export default {
...
+        } else {
+          if (result.others) {
+            result.others.identities.push(val);
+          } else {
+            result.others = {
+              name: "Others",
+              identities: [val]
+            };
+          }
+        }
```
Comment (sduenas) [inline ui/src/components/IndividualsTable.vue:130]: Wouldn't have more sense to this only once? For example (syntax can be wrong):
```
const source = val.source.toLowerCase();
const name = source;

if (!icons.find(icon => icon === source)) {
  source = "others";
  name = "Others";
}

if (result[source]) {
  result[source].identities.push(val);
} else {
  result[source] = {
    name: source,
    identities: [val]
  };
}
```

**Review by sduenas — CHANGES_REQUESTED** [review]
Please check my comments. I think they make sense for this context.

**Comment by evamillan** [comment]
They make a lot of sense, I made the changes.

**Review by sduenas — CHANGES_REQUESTED** [review]
The PR looks ok but I got some warnings running `yarn serve`:

```
/home/sduenas/devel/grimoirelab/sortinghat/ui/src/components/ExpandedIndividual.vue
   4:12  warning  Replace `·v-for="(source,·sourceIndex)·in·identities"·:key="source.name"·class="indented">` with `⏎······v-for="(source,·sourceIndex)·in·identities"⏎······:key="source.name"⏎······class="indented"`            prettier/prettier
   5:5   warning  Replace `····<v-list-item·v-for="(identity,·index)·in·source.identities"·:key="identity.uuid">` with `>⏎······<v-list-item⏎········v-for="(identity,·index)·in·source.identities"⏎········:key="identity.uuid"`  prettier/prettier
   6:3   warning  Insert `····>⏎`                                                                                                                                                                                                  prettier/prettier
   7:11  warning  Delete `··`                                                                                                                                                                                                      prettier/prettier
   8:1   warning  Replace `··············` with `············`                                                                                                                                                                     prettier/prettier
   9:11  warning  Delete `··`                                                                                                                                                                                                      prettier/prettier
```

**Review by sduenas — APPROVED** [review]
LGTM

**Pull request merged**